lib/prometheus/client/histogram.rb

```diff
@@ -89,15 +89,15 @@ def get(labels: {})
 
       # Returns all label sets with their values expressed as hashes with their buckets
       def values
-        v = @store.all_values
+        values = @store.all_values
 
-        result = v.each_with_object({}) do |(label_set, v), acc|
+        result = values.each_with_object({}) do |(label_set, v), acc|
           actual_label_set = label_set.reject{|l| l == :le }
           acc[actual_label_set] ||= @buckets.map{|b| [b.to_s, 0.0]}.to_h
           acc[actual_label_set][label_set[:le].to_s] = v
         end
 
-        result.each do |(label_set, v)|
+        result.each do |(_label_set, v)|
```

Comment:
`warning: assigned but unused variable - label_set`
I'm leaving the variable name, instead of setting to just `_` to document what params are being passed here